lsp-mode.el

```diff
@@ -6190,14 +6190,28 @@ information, for example if it doesn't support DocumentSymbols."
   :group 'lsp-imenu
   :type 'boolean)
 
+(defface lsp-details-face '((t :height 0.8 :inherit shadow))
+  "Used to display additional information troughout `lsp'.
+Things like line numbers, signatures, ... are considered
+additional information. Often, additional faces are defined that
+inherit from this face by default, like `lsp-signature-face', and
+they may be customized for finer control.")
```

Comment:
group is missing.

Reply:
Also, should we perhaps make `lsp-signature-face` inherit from `font-lock-type-face`, with a height of :0.8?

Reply:
VScode has a huge team of user experience experts and designers and for things like color selection, we follow their decisions.